### PR TITLE
Remove support for multiple BOSH Environments

### PR DESCRIPTION
**Pull request by peanball**
There is no good use case to support multiple environments and it makes implementing the boshrelease configuration unnecessarily complex.

### CODE REVIEW

File: src/pcap/.golangci.yml [hunk 1]
```diff
@@ -286,3 +285,6 @@ issues:
     - gosec
     - noctx
     - wrapcheck
+#  - path: "\\.pb\\.go"
+#    linters: []
+
```
Comment (maxmoehl) [inline src/pcap/.golangci.yml:290]: Seems like something sneaked in here that shouldn't.

**Review by maxmoehl — CHANGES_REQUESTED** [review]

File: src/pcap/.golangci.yml (hunk 1 above)
Comment (peanball) [reply src/pcap/.golangci.yml]: yes it did. fixed now.

**Review by maxmoehl — APPROVED** [review]

File: src/pcap/cmd/pcap-bosh-cli/main.go [hunk 2]
```diff
@@ -298,18 +298,17 @@ func connectToEnvironment(environmentAlias string, config *Config) (*Environment
 			return &environment, nil
 		}
 	}
-	return nil, fmt.Errorf("could not find bosh-environment %s in config", environmentAlias)
+	return nil, fmt.Errorf("could not find bosh-environment %s in BOSH CLI config", environmentAlias)
```
Comment (b1tamara) [inline src/pcap/cmd/pcap-bosh-cli/main.go:301]: we have to check if the whole function is still needed

Comment (peanball) [reply src/pcap/cmd/pcap-bosh-cli/main.go:301]: Yes because you still need to connect to the one environment.

**Pull request merged**